x/msgfees/client/cli/tx.go

```diff
@@ -35,23 +36,25 @@ func NewTxCmd() *cobra.Command {
 
 	txCmd.AddCommand(
 		GetCmdMsgFeesProposal(),
+		GetUpdateUsdConversionRateProposal(),
 	)
 
 	return txCmd
 }
 
 func GetCmdMsgFeesProposal() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "proposal [add|update|remove] [title] [description] [deposit]",
-		Args:  cobra.ExactArgs(4),
-		Short: "Submit a msg based fee proposal along with an initial deposit",
+		Use:     "proposal {add|update|remove} <title> <description> <deposit>",
+		Args:    cobra.ExactArgs(4),
+		Aliases: []string{"p"},
+		Short:   "Submit a msg based fee proposal along with an initial deposit",
 		Long: strings.TrimSpace(`Submit a msg fees proposal along with an initial deposit.
 For add, update, and removal of msg fees amount and min fee and/or rate fee must be set.
 `),
 		Example: fmt.Sprintf(`$ %[1]s tx msgfees add "adding" "adding MsgWriterRecordRequest fee"  10nhash --msg-type=/provenance.metadata.v1.MsgWriteRecordRequest --additional-fee=612nhash
-$ %[1]s tx msgfees update "updating" "updating MsgWriterRecordRequest fee"  10nhash --msg-type=/provenance.metadata.v1.MsgWriteRecordRequest --additional-fee=612000nhash 
+$ %[1]s tx msgfees update "updating" "updating MsgWriterRecordRequest fee"  10nhash --msg-type=/provenance.metadata.v1.MsgWriteRecordRequest --additional-fee=612000nhash
 $ %[1]s tx msgfees remove "removing" "removing MsgWriterRecordRequest fee" 10nhash --msg-type=/provenance.metadata.v1.MsgWriteRecordRequest
-`, version.AppName),
+		`, version.AppName),
```

Comment:
Those two tabs are inside the ticks, and as such will be included in output. And since there's no `\n` at the end, it means the next thing printed will be tabbed in.
```suggestion
`
		, version.AppName),
```